CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### App Center Distribute
 
 * **[Fix]** Fix linking framework `AuthenticationServices`.
+* **[Fix]** Fix compiling Distribute module on Xcode 12.5.
```

Comment:
It was just a warning, right? It will prevent building only with "warning as errors" setting

Reply:
Yes, but if I get it right we compile it with this flag enabled. See [Config/Global.xcconfig#L154](https://github.com/microsoft/appcenter-sdk-apple/blob/develop/Config/Global.xcconfig#L154)

Reply:
```suggestion
* **[Fix]** Fix a warning in Distribute module that prevented using SDK as a source code on Xcode 12.5.
```